src/components/elementView/powerView/powerView.tsx

```diff
@@ -0,0 +1,232 @@
+import { AssetObject, getHeaders } from "../../../utils/utils";
+import "./powerView.scss";
+
+interface PowerViewProps {
+    token: string;
+    callback?: Function;
+    asset?: AssetObject
+}
+
+interface PowerViewState {
+    powerConnections: any;
+    powerStatus: any;
+    statusLoaded: boolean;
+    alertOpen: boolean;
+    confirmationMessage: string
+}
+
+export class PowerView extends React.PureComponent<
+    RouteComponentProps & PowerViewProps,
+    PowerViewState
+    > {
+
+    //TOASTS
+    private toaster: Toaster = {} as Toaster;
+    private addToast = (toast: IToastProps) => {
+        toast.timeout = 5000;
+        this.toaster.show(toast);
+    };
+    private addSuccessToast = (message: string) => {
+        this.addToast({ message: message, intent: Intent.PRIMARY });
+    };
+    private addErrorToast = (message: string) => {
+        this.addToast({ message: message, intent: Intent.DANGER });
+    };
+
+    private refHandlers = {
+        toaster: (ref: Toaster) => (this.toaster = ref)
+    };
+
+    public state: PowerViewState = {
+        powerConnections: undefined,
+        powerStatus: undefined,
+        statusLoaded: false,
+        alertOpen: false,
+        confirmationMessage: ""
+    }
+
+    componentDidMount() {
+        axios.get(API_ROOT + "api/power/get-state/" + this.props.asset!.id, getHeaders(this.props.token))
+            .then(res => {
+                this.setState({
+                    powerConnections: res.data.power_connections,
+                    powerStatus: res.data.power_status,
+                    statusLoaded: true
+                })
+            })
+    }
+
+    render() {
+
+        return (
+            <div className={Classes.DARK}>
+                {this.state.statusLoaded ?
+                    (this.state.powerConnections === undefined ?
+                        null :
+                        Object.keys(this.state.powerConnections).map((port: string) => {
+                            return (
+                                <Card elevation={Elevation.TWO}>
+                                    <div key={port} className={"row-power"}>
+                                        <h6 className={"column-power"}>Asset Power Port: {port}</h6>
+                                        <Divider />
+                                        <h6 className={"column-power"}>
+                                            PDU Port: {Object.keys(this.state.powerConnections[port]).map((field: string) => {
+                                                return (
+                                                    this.state.powerConnections[port][field]
+                                                )
+                                            })}
+                                        </h6>
+                                        <Divider />
+                                        <h6 className={"column-power"}>{this.state.powerStatus[port]}</h6>
```

Comment:
I know this is self explanatory, but i think if the other fields have labels this should to